canonical-websites/www.ubuntu.com
# 2657: Move /about/about-ubuntu/diversity to /community/diversity

# Note
**Please merge #2685 first**

## Done

- Moved /about/about-ubuntu/diversity to /community/diversity
- Updated copy according to [copydoc](https://docs.google.com/document/d/1e85ec2AgQgLau5rrGx69cUqtCUPcD4yrrKjKHvLNM5w/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/community/diversity
- Check that it matches the [copydoc](https://docs.google.com/document/d/1e85ec2AgQgLau5rrGx69cUqtCUPcD4yrrKjKHvLNM5w/edit)
- Check that /about/about-ubuntu/diversity redirects to /community/diversity

## Issue / Card

Fixes #2657
